sdk/spring/spring-cloud-azure-stream-binder-eventhubs/src/test/java/com/azure/spring/cloud/stream/binder/eventhubs/implementation/config/ExtendedBindingHandlerMappingsProviderAutoConfigurationTests.java

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.stream.binder.eventhubs.implementation.config;
+
+import com.azure.messaging.eventhubs.LoadBalancingStrategy;
+import com.azure.spring.cloud.stream.binder.eventhubs.core.properties.EventHubsConsumerProperties;
+import com.azure.spring.cloud.stream.binder.eventhubs.core.properties.EventHubsExtendedBindingProperties;
+import com.azure.spring.cloud.stream.binder.eventhubs.core.properties.EventHubsProducerProperties;
+import com.azure.spring.messaging.eventhubs.core.checkpoint.CheckpointMode;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.autoconfigure.AutoConfigurations;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.runner.ApplicationContextRunner;
+
+import java.time.Duration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ExtendedBindingHandlerMappingsProviderAutoConfigurationTests {
+
+    private final ApplicationContextRunner contextRunner = new ApplicationContextRunner()
+        .withConfiguration(AutoConfigurations.of(EventHubsBinderConfiguration.class))
+        .withUserConfiguration(TestApp.class)
+        .withPropertyValues(
+            "spring.cloud.stream.eventhubs.default.consumer.checkpoint.mode=batch",
+            "spring.cloud.stream.eventhubs.default.consumer.load-balancing.strategy=balanced",
+            "spring.cloud.stream.eventhubs.default.consumer.batch.max-size=6",
+            "spring.cloud.stream.eventhubs.default.producer.sync=true",
```

Comment:
put these three in the consumer test case, and put the other two in the producer one.